os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1338,6 +1338,9 @@ extern void __libc_init_array(void);
 #ifdef CONFIG_PLATFORM_TIZENRT_OS
 	mpu_init();
 	app_mpu_nocache_init();
+
+	/* Initialize number of board mpu regions */
+	mpu_set_region_info(MPU_REGION_BOARD_SPECIFIC, 1);
```

Comment:
Please split the commit for this change.

Reply:
done